### PR TITLE
adding a background colour to be used with our logos

### PR DESCRIPTION
**Pull request by Nauna**
one logo we have has white font, that is intended to have a background colour.

so depending on the logo and theme, we need a background colour.

### CODE REVIEW

File: src/foam/nanos/theme/Theme.js [hunk 1]
```diff
@@ -91,6 +91,12 @@ foam.CLASS({
       name: 'largeLogoEnabled',
       documentation: 'Uses largeLogo image on various views instead of logo.'
     },
+    {
+      class: 'Color',
+      name: 'backgroundLogo',
```
Comment (jlhughes) [inline src/foam/nanos/theme/Theme.js]: Should it be called logoBackground or logoBackgrounColour?  else I would think this controls some 'background' logo.

Comment (Nauna) [reply src/foam/nanos/theme/Theme.js]: done

File: src/foam/nanos/theme/Theme.js [hunk 2]
```diff
@@ -91,6 +91,12 @@ foam.CLASS({
       name: 'largeLogoEnabled',
       documentation: 'Uses largeLogo image on various views instead of logo.'
     },
+    {
+      class: 'Color',
+      name: 'logoBackgrounColour',
```
Comment (TW80000) [inline src/foam/nanos/theme/Theme.js]: Typo

```suggestion
      name: 'logoBackgroundColour',
```

**Review by TW80000 — CHANGES_REQUESTED** [review]

**Comment by TW80000** [comment]
Shouldn't `BusinessLogoView` be updated to respect this new property as well?

**Comment by Nauna** [comment]
> Shouldn't `BusinessLogoView` be updated to respect this new property as well?

@TW80000 felt the response required some follow-up work so I moved my response to a ticket.
https://nanopay.atlassian.net/browse/CPF-3443

**Pull request merged**